examples/tuning/cta_svm/main.py

```diff
@@ -84,7 +84,7 @@ def evaluate_pipeline():
 
         wandb.finish()
 
-    pipeline_planer.wandb_sweep_agent(evaluate_pipeline, sweep_id=args.sweep_id, count=3)
+    pipeline_planer.wandb_sweep_agent(evaluate_pipeline, sweep_id=args.sweep_id, count=1)
```

Comment:
Why are you changing this to 1?

Reply:
Improve testing efficiency

Reply:
I think 3 is low enough, you want to make sure the sweep functionality is properly functioning. If count is 1, then it will only need to run once using the first setting, and do not require the obtain the another round of sweep config. I will reset to 3 to ensure this (we can discuss further if necessary).